Simplified/NYPLReaderSettings.h

```diff
@@ -21,51 +21,45 @@ typedef NS_ENUM(NSInteger, NYPLReaderSettingsFontSize) {
   NYPLReaderSettingsFontSizeLarge = 4,
   NYPLReaderSettingsFontSizeXLarge = 5,
   NYPLReaderSettingsFontSizeXXLarge = 6,
-  NYPLReaderSettingsFontSizeXXXLarge = 7
+  NYPLReaderSettingsFontSizeXXXLarge = 7,
+  NYPLReaderSettingsFontSizeLargest = 7
```

Comment:
Is this value duplication intentional?

Reply:
yeah it is. I use it to make it clear that this is meant to be the "largest" value here and so if we were to add a NYPLReaderSettingsFontSizeXXXXLarge (not that we would hopefully!), "largest" will need to be updated but it will still be semantically correct wherever it's used.   

On the other side (i.e. for "small" sizes) we have "smallest" which already expresses the above.